framework/auth/oauth_scopes.py

```diff
@@ -41,6 +41,10 @@ class CoreScopes(object):
     APPLICATIONS_READ = 'applications_read'
     APPLICATIONS_WRITE = 'applications_write'
 
+    NODE_LOG_READ = 'nodes.logs_read'
+
+    NULL = 'null'
```

Comment:
For example: I would use NULL when writing via API was totally impossible, but I would not use it as a means to hide a view from third-party usage. 

(When we compile the list of views that are writable with the admin scope, they are named explicitly- we don't just assume they are private by omission. Applications come to mind as one example)